src/mlpack/bindings/cli/get_printable_param_impl.hpp

```diff
@@ -83,13 +83,14 @@ std::string GetPrintableParam(
   const TupleType* tuple = boost::any_cast<TupleType>(&data.value);
 
   std::ostringstream oss;
-  oss << "'" << std::get<1>(*tuple) << "'";
+  oss << "'" << std::get<0>(std::get<1>(*tuple)) << "'";
 
-  if (std::get<1>(*tuple) != "")
+  if (std::get<0>(std::get<1>(*tuple)) != "")
   {
     // Make sure the matrix is loaded so that we can print its size.
-    T& mat = GetParam<T>(const_cast<util::ParamData&>(data));
-    std::string matDescription = GetMatrixSize(mat);
+    GetParam<T>(const_cast<util::ParamData&>(data));
+    std::string matDescription = std::to_string(std::get<1>(std::get<1>(*tuple))) + "x";
+                matDescription += std::to_string(std::get<2>(std::get<1>(*tuple))) + " matrix";
```

Comment:
I think either we should make the second line match the first, or you can do something like this:
```suggestion
    std::string matDescription =
        std::to_string(std::get<2>(std::get<1>(*tuple))) + "x" +
        std::to_string(std::get<1>(std::get<1>(*tuple))) + " matrix";
```
(I also had to do a few changes to make it fit in 80 characters.)

Note also that I changed it so it prints `cols x rows` not `rows x cols`.  The reason I did this is that users will typically provide data in CSV format, where the number of rows is the number of observations in the dataset.  Thus, they will be thinking about their data in row-major form.  But, inside mlpack, we use column-major form and we transpose the data on load (so, one observation == one column in C++).  I think it is better to "transpose" the size again when we print, so that the user sees the same shape as their CSV file (or whatever other input format is given). :+1:

Reply:
@rcurtin 
> (I also had to do a few changes to make it fit in 80 characters.)

Any reason for this peculiar low number?  (are you editing stuff on a phone by any chance? :)

The codebase could be more readable (on normal screens) if the limit was bumped to, say, 100 or 110 characters.

Linus Torvalds has an entirely appropriate rant about problems with the 80 character limit: https://lkml.org/lkml/2020/5/29/1038

Based on that, the linux kernel seems to have moved to 100 characters: https://www.phoronix.com/scan.php?page=news_item&px=Linux-Kernel-Deprecates-80-Col

Reply:
Actually yeah, I think a few of us do do code reviews on a phone, and it's *way* easier with the shorter lines.  But, I would say the real reason we are sticking with it is momentum---if pretty much the entire community agreed it was time to change, sure, but if there is one thing that everyone has a different opinion on... it's style.  (In my company we are trying to come up with a Julia style guide for our product, and I ended up being the "owner" of the style guide.  I learned that the 'graph of disagreements' is fully connected... e.g. every two people have some important style bit they disagree strongly on.  I regret allowing myself to end up as the "owner" of the effort...)

So, all that said, I think it's easier to mostly just stick with what we've done in the past unless there are proposals that are very widely supported (and easy to enact). :)